Expensify/App
# Add get assistance page

### Details

This PR adds a new `Get Assistance` page (shown below) which can be accessed by clicking the new question mark icon in the header of any page in the workspace settings. This page gives the option to chat with concierge or request a call with a guide. A summary of what was added/changed: 
1. Updated the header component to have the option to show the get assistance button (question mark) 
2. Updated every page of the workspace to show that button in its header
3. Added a new get assistance page, which is accessed by clicking that question mark button
4. Added constants for the various guides call Task IDs we'll associate with the guides call, depending on which page it is triggered from 
5. Added the functionality of storing a new key `lastAccessedWorkspace` in Onyx that is updated whenever a workspace is loaded (this is because there are scenarios where we can't get the workspace directly, context [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1640901164291600))
6. Removed the `InboxCallButton` component, which is no longer used anywhere in the workspace
7. Updated the request a call page to navigate back to the previous page after a call is requested, instead of always navigating to the Concierge DM
8. Updated the request a call page to use the `lastAccessedWorkspace` for the policy ID to associate with the guides job if it is available, otherwise fall back on the personal policy ID
9. Refactored the `WorkspaceSection` component to be a more generic `Section` component that can be used outside the workspace
10. Used that `Section` component on the new `GetAssistance` page and updated the `RequestCall` page to use it as well
 
### Fixed Issues

$ https://github.com/Expensify/App/issues/6719
$ https://github.com/Expensify/Expensify/issues/184190

### Tests

1. Click through every page of the workspace editor and make sure the question mark button is shown in the header. 
2. This includes every page of the bank account setup flow (info on how to test the bank setup flow [here](https://stackoverflow.com/c/expensify/questions/342/525#525)) 
3. Click that button on each page and make sure you are taken to the new Get Assistance page
4. Make sure the task ID section of the URL makes sense given the page you clicked the button from. For example, if you click it from the travel page, you should see the URL as `/get-assistance/WorkspaceBookTravel`
5. Make sure that closing the page closes the entire settings modal
6. Make sure that the back button takes you back to the page you came from
7. Make sure that clicking the `chat with concierge` option navigates you to your Concierge chat
8. Make sure that clicking the `request a setup call` option takes you to the request a call page with the same task ID in the URL (i.e. `/request-call/WorkspaceBookTravel`) 
9. Make sure that you can request a call and that you are navigated back to the page you came from after doing so
10. Check that the API call is using the proper policy ID and task ID in the Network tab


### QA Steps
1. Click through every page of the workspace editor and make sure the question mark button is shown in the header. 
2. This includes every page of the bank account setup flow
3. Click that button on each page and make sure you are taken to the new Get Assistance page
4. Make sure the task ID section of the URL (the last part) makes sense given the page you clicked the button from. For example, if you click it from the travel page, you should see the URL as `/get-assistance/WorkspaceBookTravel`
5. Make sure that closing the page closes the entire settings modal
6. Make sure that the back button takes you back to the page you came from
7. Make sure that clicking the `chat with concierge` option navigates you to your Concierge chat
8. Make sure that clicking the `request a setup call` option takes you to the request a call page with the same task ID in the URL (i.e. `/request-call/WorkspaceBookTravel`) 
9. Make sure that you can request a call and that you are navigated back to the page you came from after doing so
10. Make sure you receive an sms saying "Thanks for requesting a call with our Expensify Guides". 



### Tested On


### Screenshots


#### Web

<img width="1370" alt="Screen Shot 2022-01-03 at 2 11 15 PM" src="https://user-images.githubusercontent.com/1371996/147986065-906c09f6-683d-4a23-9b4e-583dc0d8bae8.png">

#### Mobile Web


#### Desktop

<img width="1200" alt="Screen Shot 2022-01-03 at 3 28 42 PM" src="https://user-images.githubusercontent.com/1371996/147991306-1d1a8274-c18d-4d0f-bcf6-6a612b68d888.png">

#### iOS


<img width="360" alt="Screen Shot 2022-01-03 at 3 59 03 PM" src="https://user-images.githubusercontent.com/49007721/147993047-639ffb49-cbe7-45a0-8539-9ff79186a114.png">
<img width="360" alt="Screen Shot 2022-01-03 at 3 59 05 PM" src="https://user-images.githubusercontent.com/49007721/147993051-d4b32bd9-5e09-4a3a-8f66-bf2fc0f97504.png">

#### Android

<img width="335" alt="Screen Shot 2022-01-03 at 2 35 48 PM" src="https://user-images.githubusercontent.com/1371996/147987927-628ce485-074e-4906-94fb-ddc10a4343c8.png">
<img width="324" alt="Screen Shot 2022-01-03 at 2 36 03 PM" src="https://user-images.githubusercontent.com/1371996/147987946-09bb7fb8-75d8-4bd9-ab47-68f37ed3f373.png">
